<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/logo.png</normaloff>../../img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="8">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>120</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:48pt; font-weight:704;&quot;&gt;GPX Tool&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="8">
     <widget class="QTabWidget" name="tabs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main_tab">
       <attribute name="title">
        <string>GPX</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout_left">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>
                                      &lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                      style=&quot;
                                      font-size:14pt;&quot;&gt;Files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkbox_show_gpx">
                 <property name="minimumSize">
                  <size>
                   <width>121</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>242</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Show .gpx files only</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTreeView" name="treeView"/>
               </item>
              </layout>
             </widget>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Compression settings:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QRadioButton" name="radio_button_none_compress">
                      <property name="text">
                       <string>None</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_RDP">
                      <property name="text">
                       <string>Ramer-Douglas-Peucker algorithm</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_remove_close">
                      <property name="text">
                       <string>Remove close points</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Data settings:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QCheckBox" name="checkbox_remove_GPS_errors">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Remove GPS errors</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_remove_metadata">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Remove metadata</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_remove_time">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Remove time data</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_remove_elevation">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Remove elevation data</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Plot settings:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QRadioButton" name="radio_button_none_plot">
                      <property name="text">
                       <string>None</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_elevation">
                      <property name="text">
                       <string>Plot elevation</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_speed">
                      <property name="text">
                       <string>Plot speed</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_pace">
                      <property name="text">
                       <string>Plot pace</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_ascent_rate">
                      <property name="text">
                       <string>Plot ascent rate</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radio_button_ascent_speed">
                      <property name="text">
                       <string>Plot ascent speed</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QCheckBox" name="checkbox_plot_start_stop_points">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Plot start/stop</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_plot_way_points">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Plot way points</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_plot_title">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Plot title</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_plot_colorbar">
                      <property name="minimumSize">
                       <size>
                        <width>121</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>242</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Plot colorbar</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="horizontalLayoutWidget">
               <layout class="QHBoxLayout" name="horizontalLayout_2"/>
              </widget>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout_right">
               <item>
                <layout class="QFormLayout" name="formLayout_5">
                 <item row="0" column="1">
                  <widget class="QLabel" name="progress_bar_label">
                   <property name="text">
                    <string>Parsed files</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QProgressBar" name="fileProgressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QPushButton" name="button_modify_files">
                   <property name="text">
                    <string>Modify file(s)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_plot_file">
                   <property name="text">
                    <string>Plot file</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="log">
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>
                                          &lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                          style=&quot;
                                          font-size:14pt;&quot;&gt;Log&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="info_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Read me</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExporter_les_r_sultats">
   <property name="text">
    <string>Export results</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
